udata/core/dataset/models.py

```diff
@@ -302,6 +302,8 @@ def created_at(self):
     def last_modified(self):
         if self.harvest and self.harvest.modified_at and to_naive_datetime(self.harvest.modified_at) < datetime.utcnow():
             return max([self.last_modified_internal, to_naive_datetime(self.harvest.modified_at)])
+        if self.filetype == 'remote' and self.extras.get('analysis:last-modified-at'):
+            return self.extras.get('analysis:last-modified-at')
```

Comment:
Extras should be turned to real datetime.
```suggestion
            return to_naive_datetime(self.extras.get('analysis:last-modified-at'))
```